datajoint/blob.py

```diff
@@ -504,12 +538,18 @@ def read_datetime(self):
         )
         return time and date and datetime.datetime.combine(date, time) or time or date
 
+    def read_int64_datetime(self):
+        data = self.read_value()
+        return data.astype("datetime64[us]")
+
     @staticmethod
     def pack_datetime(d):
         if isinstance(d, datetime.datetime):
             date, time = d.date(), d.time()
         elif isinstance(d, datetime.date):
             date, time = d, None
+        elif isinstance(d, np.datetime64):
```

Comment:
If datetime64 is already handled as a scalar, it does not need to be handled separately as a data structure.

Reply:
I have updated the code to reflect that.